docs/UserGuide.md

```diff
@@ -613,7 +613,7 @@ Grade a student's assignment.
 
 <div markdown="span" class="alert alert-primary">:bulb: **Extra Information**
 <br><br>
-- A student's marks need to be below the maximum marks for that particular assignment and above 0.
+- A student's marks need to between 0 (inclusive) and the maximum score assigned to the assignment (inclusive).
```

Comment:
- A student's marks need to between 0 (inclusive) and the maximum score **_of_** the assignment (inclusive).

Would the above sound better? It sounds a bit terse.

Reply:
Here would be the corrected grammar (as well as a suggested line):
-  Inputted Marks must **be** at least 0 and not exceed the maximum assignment score.

Would it sound better? Just suggesting.